### ISSUE TITLE
When turning on S3 and juicebox I can no run the enduro command.

### ISSUE THREAD

**Issue by jmelendev** [Gottwik/Enduro#49]
After setting up s3 and setting juicebox to true, i run enduro on the project and this is the message i receive:

┌————————————————————~—setting up s3—~—————————————————————┐
? Input your s3 key ********************
? Input your s3 secret key ****************************************
? What is the s3 bucket name? attic-records
? select s3 region eu-west-1
? Enable juicebox Yes
├——————————————————————————————————————————————————————————┤
│ s3 set successfully                                      │
└——————————————————————————————————————————————————————————┘
fsconlt8155:test jordan.melendez$ enduro

┌————————————————————~—Enduro started—~————————————————————┐
│ Development server started at:                           │
│     localhost:3000                                       │

│ Admin ui available at:                                   │
│     localhost:5000/admin                                 │
├——————————————————————————————————————————————————————————┤
│ Admin has no live-reload!                                │
└——————————————————————————————————————————————————————————┘
[11:12:54 | +    1.36] developer start

┌——————————————————————~—Juice pull—~——————————————————————┐
fsconlt8155:test jordan.melendez$
Note before i setup s3 the enduro function did run when juicbox was turned on. Am I missing a step that isn't documented yet?

**Comment by Gottwik** [comment]
Yea, thats not good. Could you please update to newest enduro by running `npm i enduro -g`. I added some more error info recently. Let me know if this doesn't help.

**Comment by jmelendev** [comment]
Yeah I ran the update and still no luck. It sets up s3 and juicebox fine but then when running enduro to start the project it hangs up.

```
Jordans-iMac-2:test jordanmelendez$ enduro

┌————————————————————~—Enduro started—~————————————————————┐
│ Development server started at:                           │
│     localhost:3000                                       │
│ Admin ui available at:                                   │
│     localhost:5000/admin                                 │
├——————————————————————————————————————————————————————————┤
│ Admin has no live-reload!                                │
└——————————————————————————————————————————————————————————┘
[09:47:43 | +    1.16] developer start

┌——————————————————————~—Juice pull—~——————————————————————┐
Jordans-iMac-2:test jordanmelendez$

```

**Comment by Gottwik** [comment]
I am sorry you are having this issue. I added some general logging to juicebox, could you please update enduro again and see if it prints anything out?

**Comment by jmelendev** [comment]
Ok so just ran it again this is the error that shows up.

```
┌——————————————————————~—Juice pull—~——————————————————————┐
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Key>juicebox/juice.json</Key><RequestId>D9D7C1AFA915E83C</RequestId><HostId>ZVCgqQ7Ui4089BrVrprR1pE89MSBcXnqsVySuFbvQP/zhDGZzZyWvd23GRVdYdLZXTl6lt4tgSE=</HostId></Error>
Jordans-iMac-2:test jordanmelendez$ 
```

**Comment by jmelendev** [comment]
I actually think i've gotten it to work properly. I added the response argument to the log statement you just added in which told me i had to run ```enduro juice pack -f``` first. Which worked.

**Comment by Gottwik** [comment]
Hey, good job. It's still a bug, I will take a look at it. Glad it works for you though

**Comment by jmelendev** [comment]
I hear yah. Thanks for the help though. Other than that enduro is solid, loving it so far.

**Comment by psenger** [comment]
Hello @Gottwik, I'm evaluating this product as a solution for our new website. However, I'm running into the same problem. I'm using enduro@1.4.45 globally. My s3 bucket policy is set up as a static website hosting ( I've done this a few times, and it looks correct ). What did you guys do to get this to work?

![screen shot 2018-01-09 at 1 24 45 pm](https://user-images.githubusercontent.com/211395/34702245-ce154a76-f540-11e7-804d-ed446e19448d.jpg)

**Comment by jmelendev** [comment]
i know there was an issue with specific buckets when I had to set this up. Have you tried using the EU region for your bucket? I used eu-West-1. Can your post an error code?

**Comment by psenger** [comment]
@jmelendev Thanks for your response, the bucket is empty by the way... sorry but I find it hard to believe switching regions for S3 would change anything. Is there any way you can share your policy and settings? I have this sinking feeling that this bucket needs something like versioning turned on or something. Before I go on, I have attached the screenshot. Where do I find information on JuiceBox? I cant find anything about it.

![screen shot 2018-01-09 at 4 41 34 pm](https://user-images.githubusercontent.com/211395/34706538-78133252-f55c-11e7-9b4e-200efa5861b1.jpg)
